xbmc/video/dialogs/GUIDialogVideoInfo.h

```diff
@@ -31,18 +31,18 @@ class CGUIDialogVideoInfo :
 public:
   CGUIDialogVideoInfo(void);
   virtual ~CGUIDialogVideoInfo(void);
-  virtual bool OnMessage(CGUIMessage& message);
-  virtual bool OnAction(const CAction &action);
+  bool OnMessage(CGUIMessage& message) override;
+  bool OnAction(const CAction &action) override;
```

Comment:
In the beginning I did that as well but then I changed to the opinion that it should only be there if there's also someone overriding it.